src/main/java/com/company/array/SortColors.java

```diff
@@ -0,0 +1,48 @@
+package com.company.array;
+
+public class SortColors {
+    public int[] sortColors(int[] nums) {
```

Comment:
This way looks fine. There is a straightforward way that we can calculate the count of 0, 1 and 2 in three separate variables, and accordingly we can fill the existing array which will be O(N) TC and constant space complexity.

Reply:
done